packages/appsync-emulator-serverless/dynamodbUtil.js

```diff
@@ -19,7 +20,6 @@ async function deriveDynamoDBClient(
   // start the dynamodb emulator
   const dynamoEmulator = require('@conduitvc/dynamodb-emulator');
   const dbPath = path.join(path.dirname(emulatorPath), '.dynamodb');
-  const { port } = config;
```

Comment:
This part of the function should not be using the config. This is for a default config, which is hardcoded in `config.js` as
```
module.exports = {
  DynamoDB: {
    emulator: true,
  },
};
```
So there is no port here, which means `port` will always be undefined. Instead, this line should be eliminated, and the port value passed in from the CLI should be used to launch the emulator.